aspnetcore/signalr/introduction-signalr-core.md

```diff
@@ -0,0 +1,60 @@
+---
+uid: signalr/introduction-to-signalr
+title: Introduction to SignalR on ASP.NET Core
+author: rachelappel
+ms.author: rachelap
+description: An overview of SignalR on ASP.NET Core
+manager: wpickett
+ms.date: 2/19/2018
+ms.topic: article
+ms.technology: dotnet-signalr
+ms.prod: aspnet-core
+---
+# Introduction to SignalR
+
+By [Rachel Appel](https://twitter.com/rachelappel)
+
+## What is SignalR?
+
+ASP.NET Core SignalR is a library that simplifies adding real-time web functionality to apps. Real-time web functionality is the ability to have server code push content to connected clients instantly as it becomes available, rather than having the server wait for a client to request new data.
+
+Good candidates for adding real-time updates include:
+
+* Apps that require high frequency updates from the server. Real-time gaming, social networks, maps and GPS apps, require high frequency updates.
+* Dashboards and monitoring apps. Some examples of these include voting and auction software.
+* Collaborative apps. Whiteboard apps and team meeting software are examples of collaborative apps.
+* Apps that require notifications. Social networks, email, chat, games, and many more types of apps use notifications.
+
+SignalR provides an API for creating server-to-client remote procedure calls (RPC). The RPCs call JavaScript functions on browsers or other client platforms from server-side .NET Core code. The SignalR API includes features such as connection management, groups,  messaging, and security.
+
+SignalR for ASP.NET Core:
+
+* Handles connection management automatically.
+* Enables broadcasting messages to all connected clients simultaneously. For example, a chat room.
+* Enables sending messages to specific clients or groups of clients.
+* Is open-sourced at [GitHub](https://github.com/aspnet/SignalR).
+* Scales nicely using [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/), SQL Server, or [Redis](http://redis.io).
```

Comment:
Remove the specific examples. We only support Redis right now and haven't really decided which scale-out providers to bring forward. Regardless, it would be best to not have another list to get out of date :). We can cover specific scale-out systems in a separate document on scale-out.